### PR TITLE
[wip]Updated warning messages for not verifying

### PR DESCRIPTION
**Pull request by ok236449**
Updated warning messages for not verifying email and added warning if discord isn´t verified.
Also changed icons, so the speaks for itself and you get the point of it even without reading the description.

Quick snip here:
image
(it is in czech, but you can get the idea)

### CODE REVIEW

**Comment by 1day2die** [comment]
hey,

thanks for your PR!

IMO these banners should only be shown if the Configuration "Force Discord Verification" is set to true.
I dont care about Discord on my Panel so this banner would be really annoying :)

**Comment by ok236449** [comment]
Ok, I will make it only when set as forced way of verification and request pull again.

**Comment by 1day2die** [comment]
also make sure to create branches from the Dev-Branch of the CP :)

**Comment by ok236449** [comment]
> also make sure to create branches from the Dev-Branch of the CP :)

Okay, sorry

**Comment by ok236449** [comment]
Can you please look at the last commit named "Why won't this work?"

I don´t know what I am doing wrong here.
(don´t want to waste your time, I guess you would have done it in a matter of seconds, but I am trying to learn this language(php) from the ground up.

File: resources/views/layouts/main.blade.php [hunk 1]
```diff
@@ -297,18 +297,32 @@ class="nav-link @if(Request::routeIs('admin.settings.*')) active @endif">
...
+            @else
+                <div class="alert alert-warning p-2 m-2">
+                    <h5><i class="fas fa-envelope"></i> Warning!</h5>
+                    Verification link has been sent to your email.<br>
+                    Please contact support If you didn't receive your verification email.
+                </div>
             @endif
+            
+        @endif
+        @if(is_null(Auth::user()->discordUser))
```
Comment (AVMG20) [inline resources/views/layouts/main.blade.php:319]: this if statement is not conditional, not all hosts see a need in requiring discord verification

**Comment by 1day2die** [comment]
changed title to [wip] prefix for the time being :)

**Comment by 1day2die** [comment]
Going to close this PR

Thanks for you contribution, but in the meantime this has become so outdated^^

**Pull request closed without merging**